docs/pages/admin-guides/deploy-a-cluster/helm-deployments/ibm.mdx

```diff
@@ -0,0 +1,325 @@
+
+  secret/teleport-tls created
+  ```
+- Create the secret containing the bucket credentials
+  ```code
+  $ kubectl -n teleport-cluster create secret generic bucket-creds \
+    --from-literal=AWS_ACCESS_KEY_ID=<put the key id> \
+    --from-literal=AWS_SECRET_ACCESS_KEY=<put the secret access key here>
+
+  secret/bucket-creds created
+  ```
+- Create the configmap containing the PostgreSQL CA certificate
+  ```code
+  $ kubectl -n teleport-cluster create configmap postgres-ca \
+    --from-file=cacert.pem=postgres-certs.pem
+
+  configmap/postgres-ca created
+  ```
+- Create the secret containing the PostgreSQL password
+  ```code
+  $ kubectl -n teleport-cluster create secret generic postgres-pass \
+    --from-literal=password="$POSTGRES_PASSWORD"
+
+  secret/postgres-pass created
+  ```
+
+## Step 5/7. Configure and deploy Teleport
+
+In this step we will deploy a Teleport cluster with the `teleport-cluster` Helm chart.
+
+Create the following `values.yaml` file and replace the placeholders:
+```yaml
+chartMode: standalone
+clusterName: teleport.example.com # replace this with your cluster domain
+persistence:
+  enabled: false
+
+auth:
+  teleportConfig:
+    teleport:
+      storage:
+        type: postgresql
+        # replace the database address and port with the values you saved during the PostgreSQL creation
+        # the result should look like: "postgres://teleport@60d25e94-edf5-4494-a6cc-5fcae28aa7c2.c0v4phir0ah9ul9trho0.databases.appdomain.cloud:31766/teleport_backend?sslmode=verify-full"
+        conn_string: postgres://teleport@<DATABASE ADDRESS>:<DATABASE PORT>/teleport_backend?sslmode=verify-full
+        # replace the database address and port with the values you saved during the PostgreSQL creation
+        change_feed_conn_string: postgresql://repl@<DATABASE ADDRESS>:<DATABASE PORT>/teleport_backend?sslmode=verify-full
+        audit_events_uri:
+        # replace the database address and port with the values you saved during the PostgreSQL creation
+          - "postgres://teleport@<DATABASE ADDRESS>:<DATABASE PORT>/teleport_audit?sslmode=verify-full#disable_cleanup=false&retention_period=8766h"
+        # replace the bucket name and endpoint with the values you saved during the bucket creation
+        audit_sessions_uri: 's3://<BUCKET_NAME>/readonly/records?endpoint=<DIRECT ENDPOINT>&region=ibm'
+
+  extraVolumes:
+    - name: postgres-ca
+      configMap:
+        name: postgres-ca
+  extraVolumeMounts:
+    - name: postgres-ca
+      mountPath: /var/lib/postgres-ca
+      readOnly: true
+
+  extraEnv:
+    - name: AWS_ACCESS_KEY_ID
+      valueFrom:
+        secretKeyRef:
+          name: bucket-creds
+          key: AWS_ACCESS_KEY_ID
+          optional: false
+    - name: AWS_SECRET_ACCESS_KEY
+      valueFrom:
+        secretKeyRef:
+          name: bucket-creds
+          key: AWS_SECRET_ACCESS_KEY
+          optional: false
+    - name:  PGSSLROOTCERT
+      value: /var/lib/postgres-ca/cacert.pem
```

Comment:
I'd rather have this passed in as `sslrootcert` parameter in the specific conn strings (not that the auth server should ever be opening other postgres connections, but you never know).